examples/tailwind/tailwind.config.js

```diff
@@ -1,5 +1,8 @@
 module.exports = {
-  purge: ["./pages/**/*.{js,jsx,ts,tsx}", "./components/**/*.{js,jsx,ts,tsx}"],
+  purge: [
+    "./app/pages/**/*.{js,jsx,ts,tsx}",
+    "./app/components/**/*.{js,jsx,ts,tsx}"
+  ],
```

Comment:
Thanks! One tweak:

```suggestion
  purge: [
    "./**/{pages,components}/**/*.{js,jsx,ts,tsx}",
  ],
```

I'm not sure if this actually works, but it should be something close to this. Match file inside any page or component directory anywhere inside the project, including at root.